README.md

```diff
@@ -272,6 +272,29 @@ module.exports = {
 }
 ```
 
+## Using svelte-loader in combination with thread-loader
```

Comment:
Would this feature be useful in other cases? (e.g. I think you mentioned tailwind and babel). If so, can we describe this feature more generically? I'm find mentioning `thread-loader`, but it maybe shouldn't be the focus

Reply:
I can't think of other cases... It is mainly useful because it allows other processess/workers to get css reliably, and not try to access an empty Map.
The default Map storage with file paths in a query will only be accessible in that exact process.
@trash-and-fire told me that we can detect if `thread-loader` is used in a pipeline with svelte and turn this on automatically.
Maybe we should make it that way.. :thinking:

Reply:
That might be nicer than adding an option. But I also wonder if it's even worth adding something that's only useful for `thread-loader`

Reply:
Part of the reason I'm here is because one day one man tried to use `svelte-loader` with `svelte-preprocess` and had 4 mins of build-time and 40 seconds of reload time for 20 or so components. :grin: 
That would save him.

Reply:
Does that mean that you need to use `thread-loader` to deal with https://github.com/sveltejs/svelte-preprocess/issues/275 ?

Reply:
Exactly. It will speed up that thing significantly.
But you also need to configure preprocess and webpack `.css` pipeline correctly to gain maximum performance.

Reply:
I think this is not an entirely accurate statement.

There was a problem of rebuilding all virtual css files if only one changed. This issue has been fixed without using a thread loader. A thread loader can provide additional performance by parallelizing the compilation of each file into a process pool. But due to the lack of shared memory between processes, we must transfer all the content of the css file through the import line.